Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:8
+
+# Install the OSS Review Toolkit
+ENV APPDIR=/opt/oss-review-toolkit
+COPY . "${APPDIR}"
+WORKDIR "${APPDIR}"
+RUN ./gradlew installDist
+
+# Install dependencies for NPM scanning
+RUN ["/bin/bash", "-c", "set -o pipefail && curl -sL https://deb.nodesource.com/setup_8.x | bash -"]
```

Comment:
You might want to pass `-o pipefail` directly to `bash`.

Reply:
Thanks, didn't know I could do that.

Reply:
Fixed

Reply:
Also, we're having quite a few `RUN` statements here. AFAIK, each `RUN` [adds a new Docker layer](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run), and we might want to avoid too many layers, e.g. by putting all `/bin/bash` calls into a single layer.

Reply:
I was going to but I wasn't sure I could because of the piping... Relatively new bash user... What would this look like?

Reply:
I believe in total the line should look like:

    RUN ["/bin/bash", "-o", "pipefail", "-c", "curl -sL https://deb.nodesource.com/setup_8.x | bash - && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && echo \"deb https://dl.yarnpkg.com/debian/ stable main\" | tee /etc/apt/sources.list.d/yarn.list"]